### PR TITLE
add-type-column-to-tags-1982

### PR DESCRIPTION
**Pull request by Markiyan000**

### CODE REVIEW

**Review by forestik — APPROVED** [review]

File: core/src/test/java/greencity/validator/TipsAndTricksDtoRequestValidatorTest.java [hunk 1]
```diff
@@ -24,13 +25,15 @@
     @Mock
     private TagsService tagService;
 
+    private static final TagType TAG_TYPE = TagType.TIPS_AND_TRICKS;
```
Comment (vterlyha) [inline core/src/test/java/greencity/validator/TipsAndTricksDtoRequestValidatorTest.java:28]: As I see, you can just use TagType.TIPS_AND_TRICKS as parameter in method. No need to create additional field

File: service/src/main/java/greencity/service/EcoNewsServiceImpl.java [hunk 2]
```diff
@@ -323,7 +328,7 @@ public void update(EcoNewsDtoManagement ecoNewsDtoManagement, MultipartFile imag
         toUpdate.setTitle(ecoNewsDtoManagement.getTitle());
```
Comment (vterlyha) [inline service/src/main/java/greencity/service/EcoNewsServiceImpl.java:328]: Move all settings to private method - smth like enhanceWithManagementData ...

File: service/src/main/java/greencity/service/EcoNewsServiceImpl.java [hunk 3]
```diff
@@ -344,7 +349,8 @@ public EcoNewsDto update(UpdateEcoNewsDto updateEcoNewsDto, MultipartFile image,
         toUpdate.setTitle(updateEcoNewsDto.getTitle());
         toUpdate.setText(updateEcoNewsDto.getText());
         toUpdate.setSource(updateEcoNewsDto.getSource());
-        toUpdate.setTags(modelMapper.map(tagService.findTagsByNames(updateEcoNewsDto.getTags()),
+        toUpdate.setTags(modelMapper.map(tagService
+                .findTagsByNamesAndType(updateEcoNewsDto.getTags(), TagType.ECO_NEWS),
```
Comment (vterlyha) [inline service/src/main/java/greencity/service/EcoNewsServiceImpl.java:353]: Same here

File: service/src/main/java/greencity/service/TipsAndTricksServiceImpl.java [hunk 4]
```diff
@@ -71,7 +72,8 @@ public TipsAndTricksDtoResponse save(TipsAndTricksDtoRequest tipsAndTricksDtoReq
         if (image != null) {
             toSave.setImagePath(fileService.upload(image).toString());
         }
-        toSave.setTags(modelMapper.map(tagService.findTagsByNames(tipsAndTricksDtoRequest.getTags()),
+        toSave.setTags(modelMapper.map(tagService
```
Comment (vterlyha) [inline service/src/main/java/greencity/service/TipsAndTricksServiceImpl.java:75]: Also here and method down

**Review by KhonkoDmytro — APPROVED** [review]

**Review by SashkoMolodec — APPROVED** [review]

**Pull request merged**